src/common/options/osd.yaml.in

```diff
@@ -507,32 +507,77 @@ options:
     stats (inc. scrub/block duration) every this many seconds.
   default: 120
   with_legacy: false
-# when replicas are slow to respond to scrub resource reservations
-# Note: disable by using a very large value
 - name: osd_scrub_slow_reservation_response
   type: millisecs
   level: advanced
-  desc: Duration before issuing a cluster-log warning
-  long_desc: Waiting too long for a replica to respond (after at least half of the
-    replicas have responded).
+  desc: Maximum wait (milliseconds) for scrub reservations before issuing a cluster-log warning
+  long_desc: Waiting too long for a replica to respond to scrub resource reservation request
+   (after at least half of the replicas have responded). Disable by setting to a very large value.
   default: 2200
   min: 500
   see_also:
   - osd_scrub_reservation_timeout
   with_legacy: false
-# when a replica does not respond to scrub resource request
-# Note: disable by using a very large value
 - name: osd_scrub_reservation_timeout
   type: millisecs
   level: advanced
-  desc: Duration before aborting the scrub session
-  long_desc: Waiting too long for some replicas to respond to
-    scrub reservation requests.
+  desc: Maximum wait (milliseconds) for replicas' response to scrub reservation requests
+  long_desc: Maximum wait (milliseconds) for all replicas to respond to
+    scrub reservation requests, before the scrub session is aborted. Disable by setting
+    to a very large value.
```

Comment:
The changes here appear to be applicable to the current implementation as well, should be in their own commit.

Reply:
Most of the changes are related to this PR. I have now extract the rest into https://github.com/ceph/ceph/pull/53040